Layout/Shared/Notification/constants.js

```diff
@@ -0,0 +1,10 @@
+export const OPTIMIZER = "optimizer";
+export const NOTIFICATION_TYPE = "localStorage";
+
+export const messageForDissmissNotification = {
+  optimizer: {
+    name: "Optimizer Pools",
+    description:
+      "Some exciting is on optimizer pools, please check on the compound and earn page!",
```

Comment:
This description sounds odd, did bigwampa provided the text?

Reply:
Need to work on content, going to check the adjacent ticket assign to Ncookie for content 

> This description sounds odd, did bigwampa provided the text?